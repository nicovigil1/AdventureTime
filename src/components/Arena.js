import React from 'react'
import Spritesheet from 'react-responsive-spritesheet';

export default class Arena extends React.Component {
  render() {
    return(
      <div style={styleFlex}>
          <Spritesheet
            id="player"
            style={stylePlayer}
            image={`https://i.ibb.co/Kjd23z4/adventurer-1-3-Sheet.png`}
            widthFrame={50}
            heightFrame={37}
            steps={4}
            fps={6}
            autoplay={true}
            loop={true} />
          <Spritesheet
            id="enemy"
            style={styleEnemy}
            image={`https://i.ibb.co/jyBGFQJ/Heavy-Bandit-Spritesheet.png`}
            widthFrame={48}
            heightFrame={48}
            steps={4}
            fps={6}
            autoplay={true}
            loop={true} />
      </div>
    )
  }
}

const styleEnemy = {  
  height: "200px", 
  width: "200px",
  position: "relative",
  right: "13vw"
}

const stylePlayer = {  
  height: "200px", 
  width: "200px",
  alignSelf: "flex-end",
  position: "relative",
}

const styleFlex = {
  display: "flex",
  flexDirection: "row",
  alignSelf: "flex-end",
  width: "100vw"
}