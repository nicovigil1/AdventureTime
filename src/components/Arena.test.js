import React from 'react';
import Spritesheet from 'react-responsive-spritesheet';
import Arena from './Arena';
import { shallow } from 'enzyme';
import { isMainThread } from 'worker_threads';

describe('Header', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Arena />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})