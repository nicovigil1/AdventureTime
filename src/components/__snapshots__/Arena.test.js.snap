// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<div
  style={
    Object {
      "alignSelf": "flex-end",
      "display": "flex",
      "flexDirection": "row",
      "width": "100vw",
    }
  }
>
  <t
    autoplay={true}
    background=""
    backgroundPosition=""
    backgroundRepeat="no-repeat"
    backgroundSize="cover"
    className=""
    direction="forward"
    endAt={false}
    fps={6}
    getInstance={[Function]}
    heightFrame={37}
    id="player"
    image="https://i.ibb.co/Kjd23z4/adventurer-1-3-Sheet.png"
    isResponsive={true}
    loop={true}
    onClick={[Function]}
    onDoubleClick={[Function]}
    onEachFrame={false}
    onEnterFrame={false}
    onInit={[Function]}
    onLoopComplete={false}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseMove={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onMouseUp={[Function]}
    onPause={[Function]}
    onPlay={[Function]}
    onResize={false}
    startAt={0}
    steps={4}
    style={
      Object {
        "alignSelf": "flex-end",
        "height": "200px",
        "position": "relative",
        "width": "200px",
      }
    }
    timeout={0}
    widthFrame={50}
  />
  <t
    autoplay={true}
    background=""
    backgroundPosition=""
    backgroundRepeat="no-repeat"
    backgroundSize="cover"
    className=""
    direction="forward"
    endAt={false}
    fps={6}
    getInstance={[Function]}
    heightFrame={48}
    id="enemy"
    image="https://i.ibb.co/jyBGFQJ/Heavy-Bandit-Spritesheet.png"
    isResponsive={true}
    loop={true}
    onClick={[Function]}
    onDoubleClick={[Function]}
    onEachFrame={false}
    onEnterFrame={false}
    onInit={[Function]}
    onLoopComplete={false}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseMove={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onMouseUp={[Function]}
    onPause={[Function]}
    onPlay={[Function]}
    onResize={false}
    startAt={0}
    steps={4}
    style={
      Object {
        "height": "200px",
        "position": "relative",
        "right": "13vw",
        "width": "200px",
      }
    }
    timeout={0}
    widthFrame={48}
  />
</div>
`;
